prowler/providers/common/audit_info.py

```diff
@@ -46,10 +46,8 @@ def print_gcp_credentials(self, audit_info: GCP_Audit_Info):
         # Beautify audited profile, set "default" if there is no profile set
         try:
             getattr(audit_info.credentials, "_service_account_email")
```

Comment:
Why this? I think now is no longer needed since the `getattr` is done in the next line.